nrich-registry/src/main/java/net/croz/nrich/registry/data/constant/RegistryDataConstants.java

```diff
@@ -31,6 +35,14 @@ public final class RegistryDataConstants {
 
     public static final String REGISTRY_FORM_ID_UPDATE_SUFFIX = "update";
 
+    public static final String REQUEST_PACKAGE_NANE = "request";
+
+    public static final String PACKAGE_SEPARATOR = "\\.";
+
+    public static final int LAST_PACKAGE_MARKER_INDEX = 2;
+
+    public static final List<String> SUFFIX_LIST_TO_REMOVE = Collections.unmodifiableList(Arrays.asList("model", "entity"));
```

Comment:
Name of this constant is to generic. It sounds like we are always removing these values but what we are actually doing is replacing one of these values with `"request"` value when we are constructing class name for request package.

Reply:
New name is better but I'm still not sure if it's clear enough. I 'll try to find something and let you know.

Reply:
Maybe we can solve this by adding javadoc. It would be easier to understand with some description.